North-Seattle-College/ad440-winter2022-thursday-repo
# Feature/handle errors

#### Description of change
- The text "Comment Feedback" only shows when a valid response (200) is returned.
- If an error is returned from the API,  a user-friendly error message will show. 

#### Please Note:  Due to the current API not being fully functional. A test API will need to be used to test the error response. 
- Test API URL: https://jsonplaceholder.typicode.com/todos/1

#### How to test
- Clone the repo
- cd frontend
- npm install
- npm start
- git checkout feature/handle-errors
- You should see the below UI:
![Screen Shot 2022-03-16 at 9 34 50 PM](https://user-images.githubusercontent.com/47171828/158737946-7e56c8ed-e2b7-4ebd-ae4f-8b5c9bee1699.png)
- Write a comment in the comment box and "Submit"
- You should see the below "Comment Feedback" box:
![Screen Shot 2022-03-16 at 9 33 02 PM](https://user-images.githubusercontent.com/47171828/158738702-054ec2f9-0bfb-405b-aab7-2a5578ff3535.png)
- In /ad440-winter2022-thursday-repo/frontend/src/components/SubmitBtn/SubmitBtn.js on line 4. Replace the URL with " https://jsonplaceholder.typicode.com/todos/1" and save the file. 
- Write a comment in the comment box and "Submit"
- You should see the below "Error" box:
![Screen Shot 2022-03-16 at 8 48 59 PM](https://user-images.githubusercontent.com/47171828/158739237-f13b874f-6480-45eb-b2e0-8fa5596abef8.png)
-  Check the console for the below error response.

![Screen Shot 2022-03-16 at 8 53 40 PM](https://user-images.githubusercontent.com/47171828/158739293-72b0f52d-9063-48e0-9695-2e3a55c31154.png)
